Libraries/Text/TextInput/Multiline/RCTMultilineTextInputView.h

```diff
@@ -11,6 +11,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface RCTMultilineTextInputView : RCTBaseTextInputView
 
+#if TARGET_OS_OSX
+@property (nonatomic, assign) BOOL scrollEnabled;
+#endif
+
```

Comment:
We mark macOS specific changes with the following tags to make it easy to see where we have introduced diffs with RN Core. Could you add those here as well?
```suggestion
#if TARGET_OS_OSX // [TODO(macOS GH#774)
@property (nonatomic, assign) BOOL scrollEnabled;
#endif // TODO(macOS GH#774)]

```